Signal/src/environment/iOSVersions.h

```diff
@@ -9,94 +9,6 @@
 #import <Foundation/Foundation.h>
 #include <Availability.h>
 
-// Source: https://github.com/carlj/CJAMacros/blob/master/CJAMacros/CJAMacros.h
+#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(major, minor) \
```

Comment:
AFAIK you only ever need a single method. I wanted to move away from the existing macros since they use constant-ized iOS versions and the constants are incomplete and non-trivial to update. Far simpler (and just as readable) to pass the major minor versions in as ints.